.github/workflows/frontend.yml

```diff
@@ -48,12 +48,21 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
+
       - name: yarn install
         run: yarn install --frozen-lockfile
-      - run: yarn lint
-      - run: yarn build
-      - run: yarn test
+
+      - name: lint
+        run: yarn lerna -- run lint --since origin/master
```

Comment:
this one is `yarn lerna -- run lint`, the below is `yarn lerna run lint` - i'm not clear on what the double dashes would be for here. Same goes for all the places further down, it varies for some reason.

Reply:
The first set of `--` is for yarn, the second is for lerna. Not really needed everywhere but added it for consistency. Fixed some inconsistencies.